target/linux/ath79/image/generic.mk

```diff
@@ -101,6 +101,19 @@ define Build/wrgg-pad-rootfs
 	$(STAGING_DIR_HOST)/bin/padjffs2 $(IMAGE_ROOTFS) -c 64 >>$@
 endef
 
+define Build/engenius_ens202ext-factory
+	-[ -f "$@" ] && \
```

Comment:
Can someone tell me what this line does. I can't find any reference anywhere. I simply copied it from #2299